File: PROJECT/FRONTEND/Common/DTOs/ShippingLabelDTO.ts


export class ShippingLabelDTO { 
    constructor(obj: Partial<ShippingLabelDTO>) {
        Object.assign(this,obj);
    }

    
    public senderClient: ClientDTO;
    public receiverClient: ClientDTO;
    public senderAddress: AddressDTO;
    public receiverAddress: AddressDTO;
    public packCount: number;
    public shipmentType: string;
    public weight: number;
    public shipmentDescription: string;
}

File: PROJECT/Models/DTOs/Shipping/Econt/ShippingLabelDTO.cs
namespace Models.DTOs.Shipping.Econt
{
    public class ShippingLabelDTO
    {
        public ClientDTO SenderClient { get; set; }
        public ClientDTO ReceiverClient { get; set; }
        public AddressDTO SenderAddress { get; set; }
        public AddressDTO ReceiverAddress { get; set; }
        public int PackCount { get; set; }
        public string ShipmentType { get; set; }
        public int Weight { get; set; }
        public string ShipmentDescription { get; set; }
    }
}
